Database/Tool	Version

GAIA	v3.5.3

Reference genome	GRCh38 / HG38

Ensembl VEP Cache	110

GENCODE (VEP)	44 (43 for coverage calculation)

RefSeq (VEP)	110 (GCF_000001405.40_GRCh38.p14_genomic.gff)

PolyPhen (VEP)	2.2.3

SIFT (VEP)	6.2.1

dbSNP (VEP)	154

COSMIC (VEP)	97

COSMIC (GenomeScreensMutant&NonCodingVariants)	v99

gnomAD exomes (VEP)	r2.1.1, exomes only (remapped)

gnomAD genomes (VEP)	r3.1.2, genomes only

gnomAD hemizygote exomes (VEP)	r2.1 (remapped)

gnomAD hemizygote genomes	v3.1.1

ClinPred	hg19 (remapped)

dbNSFP	4.4a (gnomad_exomes_nhomalt, gnomad_genomes_nhomalt, VEST4, MutPred2, BayesDel_noAF)

OMIM	20240707

MANE	GRCh38r1.2

Appris (for coverage calculation)	2023_08.v48

ClinVar	20240630

CADD	1.6 (VEP Plugin 110)

REVEL	1.3 (VEP Plugin 110)

AlphaMissense	hg38 (VEP Plugin 110)

pLI/LOF	gnomAD 2.1.1 (VEP custom 110)

HGMD	2023Q4

MasterMind	2024.01.03

SpliceAI	genome_scores_v1.3

NovaSeq Control Cohort	136 Samples (66 Male, 70 Female; 71 Mouth swab, 64 Blood, 1 DNA)

TWIST Baseline Cohort	79 Samples (37 Male, 42 Female; 71 Mouth swab, 7 Blood, 1 DNA)

WES CNV OMIM 2020.10.20

WES CNV DGV_1kG	2020.02.05

WES CNV clinvar	2021.06.02

WES CNV gnomAD SVs	2021.06.02

WES CNV hgmd	2020Q3

HGNC nomenclature	2023-09-15 16:32

WES CNV annotation	VEP103

Ancestry Estimate	somalier v0.2.19 (sites file gnomad.exomes.r2.1 were created from 1kg 2019-02-19)

contamination estimation	VerifyBamID2 Version:2.0.1 (marker files were created from 1kg 2020-08-05)

Variant Calling Assessment	RTG Tools 3.12.1
