## NSWHP Genomic Annotation Interpretation Application (GAIA) and GenPhen Atlas Inverntory List

### [GenPhen Atlas](http://192.168.106.150:8000/)

*  For trial only, include 5000 samples sequenced on NovaSeq6000 from 2020 to early 2022

### GAIA 3.6.0 - for NovaSeq GRCh38/hg38 (LIVE TEST ONLY! PLEASE REPORT BUGS TO [BIOINFORMATICS TEAM](NSWPATH-BioinformaticsEAST@health.nsw.gov.au)) [What's New](./another-page_3.6.0.html)?

[LAB PRODUCTION - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.6.0)](http://192.168.106.151:5878/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.6.0)](http://pilot.gaia.nswhp.com.au:5878/)

For databases: 
*     LIVE TEST ONLY! DON'T USE FOR DIAGNOSTIC PURPOSE UNITL RELEASE!

### GAIA 3.5.3 - for NovaSeq GRCh38/hg38 (release date 20240723) [What's New](./another-page_3.5.3.html)?

[LAB PRODUCTION - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.3)](http://192.168.106.151:5877/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.3)](http://pilot.gaia.nswhp.com.au:5877/)

For databases: 
*     PTW_24_0701A onwards

### GAIA 3.5.2 - for NovaSeq GRCh38/hg38 (release date 20240110) [What's New](./another-page_3.5.2.html)?

[LAB PRODUCTION - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.2)](http://192.168.106.151:5875/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.2)](http://pilot.gaia.nswhp.com.au:5875/)

For databases: 
*     PTW_24_0101A onwards
*     Training WES cases
*     WES re-analysis
*     WGS re-analysis

### GAIA 3.5.1 - for NovaSeq GRCh38/hg38 (release date 20231010) [What's New](./another-page_3.5.1.html)?

[LAB PRODUCTION - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.1)](http://192.168.106.151:5874/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.1)](http://pilot.gaia.nswhp.com.au:5874/)

For databases: 
*     PTW_23_1002A onwards
*     WES re-analysis
*     WGS re-analysis
*     RNSH in PTW_23_0814A

### GAIA 3.5.0 - for NovaSeq GRCh38/hg38 (release date 20230918) [What's New](./another-page_3.5.0.html)?

[LAB for WES only - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.0)](http://192.168.106.151:5873/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.5.0)](http://pilot.gaia.nswhp.com.au:5873/)

For databases: 
*     PTW_23_0807B onwards
*     WES re-analysis

### GAIA 3.3.15 - for NovaSeq GRCh38/hg38 (release date 20230822) [What's New](./another-page_3.3.15.html)?

[LAB for WES only - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.3.15)](http://192.168.106.151:5872/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.3.15)](http://pilot.gaia.nswhp.com.au:5872/)

For databases: 
*     PTW_23_0807A onwards
*     WES re-analysis

### GAIA 3.3.13 - for NovaSeq GRCh38/hg38 (release date 20230124) [What's New](./another-page_3.3.13.html)?

[LAB for WES only - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.3.13)](http://192.168.106.151:5871/)

[CLOUD DEVELOPMENT - Genomics Annotation Interpretation Analysis pipeline (GAIA 3.3.13)](http://pilot.gaia.nswhp.com.au:5871/)

For databases: 
*     PTW_23_0116A onwards
*     Duplicated PTW_22_1125A, PTW_22_1130A, PTW_22_1205A, PTW_22_1212A, PTW_22_1212B, PTW_22_1219A, PTW_22_1219B_2, PTW_23_0102A, PTW_23_0102B and PTW_23_0109A as Joint Call not bringing back low quality variants that present in single sample VCF files.
*     WGS & WES re-analysis
*     RNSH in PTW_23_0327A

### POWH BaseSpaces WorkGroup

[NSWHP Private Domain BaseSpace](https://nswhp-pl.aps2.sh.basespace.illumina.com/)
Once log in, on top right corner, choose POWH (green tick) for workgroup, choose Personal for testing. 
Java Exception URL is `https://basespace-launch-east.s3.amazonaws.com`

### IGV configuration on PCs

In some circumstance, IGV genome build cannot be loaded due to the proxy. So, we need to configure IGV to use genome build list stored on health web server. 
The instruction guides of [How to setup IGV for local hosted Genomes](./How_to_setup_IGV_for_local_hosted_Genomes.html)

